tasks/apply-mapping/apply-mapping.yaml

```diff
@@ -86,6 +89,13 @@ spec:
                 variable=$(jq -c --argjson i "$i" '.[$i]' <<< "${SUPPORTED_VARIABLES}")
                 KEY=$(jq -r 'to_entries[] | .key' <<< "$variable")
                 VALUE=$(jq -r 'to_entries[] | .value' <<< "$variable")
+                # If a tag is going to be replaced by a null value, error out
```

Comment:
It's probably good to have this check. But I can't think of any way this could happen. Can you? But maybe we saw a -null tag generated at some point? Not sure.

Reply:
Now I see that you explicitly set timestamp to null if we can't get the build-date. So yeah, I guess it makes sense. But my question still stands.

Reply:
This can also happen to git_sha if `source.git.revision` is not there in the component. I *think* someone hit that a long time ago

Reply:
ok